src/pages/code-of-conduct.js

```diff
@@ -0,0 +1,32 @@
+import React, { useRef } from "react"
+
+import SEO from "../components/SEO/SEO"
+import Nav from "../components/Nav/Nav"
+import Footer from "../components/Footer/Footer"
+import CodeofConduct from "../components/DocPages/CodeofConduct"
+
+const scrollTo = ref => {
+  if (!ref || !ref.current) {
+    return
+  }
+  window.scrollTo(0, ref.current.offsetTop - 100)
+}
+
+const CodeofC = () => {
```

Comment:
for other pages, we've been adding `Page` as a suffix for page components, like `IndexPage`. would be nice to stay consistent

also we're using pascal case for component names, so would be `CodeOfC`